packages/react/src/components/Authenticator/ConfirmSignIn/ConfirmSignIn.tsx

```diff
@@ -29,25 +33,27 @@ export const ConfirmSignIn = (): JSX.Element => {
   } = useCustomComponents();
 
   return (
-    <form
-      data-amplify-form=""
-      data-amplify-authenticator-confirmsignin=""
-      method="post"
-      onChange={handleChange}
-      onSubmit={handleSubmit}
-    >
-      <Flex as="fieldset" direction="column" isDisabled={isPending}>
-        <Header />
+    <RouteContainer className={className} variation={variation}>
```

Comment:
For the `route` component files, this PR just wraps the components in a `RouteContainer` component that consists of the HTML related elements that were previously inside the `Router`